java/code/src/com/redhat/rhn/manager/system/SystemManager.java

```diff
@@ -3933,7 +3934,7 @@ public static void setReportDbUser(MinionServer minion, boolean forcePwChange) {
         }
         Credentials credentials = Optional.ofNullable(mgrServerInfo.getReportDbCredentials())
                 .orElse(CredentialsFactory.createCredentials(
-                        "hermes_" + RandomStringUtils.random(8, "abcdefghijklmnopqrstuvwxyz"),
+                        "hermes_" + RandomStringUtils.random(8, 0, 0, true, false, null, new SecureRandom()),
```

Comment:
Not sure about this. We just want lowercase characters. Security is not important here as we generate a (unique) username and not a password.

Reply:
I can lowercase the generated result... or ignore that warning, indeed

Reply:
oops, I forgot that I still needed to fix this one... I'll prepare a follow up PR